Sources/RegexBuilder/Algorithms.swift

```diff
@@ -49,8 +49,8 @@ extension BidirectionalCollection where SubSequence == Substring {
   /// - Returns: `true` if the regex returned by `content` matched anywhere in
   ///   this collection, otherwise `false`.
   @available(SwiftStdlib 5.7, *)
-  public func contains<R: RegexComponent>(
-    @RegexComponentBuilder _ content: () -> R
+  public func contains(
+    @RegexComponentBuilder _ content: () -> some RegexComponent
   ) -> Bool {
     contains(content())
   }
```

Comment:
Unfortunately I believe these will fail to integrate with the current Swift compiler due to rdar://92459215, see the comment at the top of the file (I don't know if `some RegexComponent<Output>` works tho)

Reply:
Opened https://github.com/apple/swift-experimental-string-processing/pull/458 to verify

Reply:
Hmm, seems to be fixed now (other test failure unrelated)